{"name": "创建英雄失败的测试用例-(攻+1)", "status": "passed", "parameters": [{"name": "success_for_name", "value": "'123@qq.com'"}, {"name": "success_for_volume", "value": "99"}, {"name": "fail_for_power", "value": "0"}], "start": 1684150564922, "stop": 1684150564922, "uuid": "8e617452-73d8-4454-b0cc-ad42a3feb451", "historyId": "198e5a2dd3e025c098eecd28dca4b7fc", "testCaseId": "e6505b56913fd0a5b5c514cb9e6b3fa7", "fullName": "test_create_hero_homework_01.TestHeroCreate#test_create_hero_fail_for_power_p1", "labels": [{"name": "tag", "value": "run(order=7)"}, {"name": "suite", "value": "test_create_hero_homework_01"}, {"name": "subSuite", "value": "TestHeroCreate"}, {"name": "host", "value": "LAPTOP-0EHG7NSH"}, {"name": "thread", "value": "48660-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_create_hero_homework_01"}]}